packages/expo-symbols/expo-module.config.json

```
@@ -0,0 +1,9 @@
+{
+  "platforms": ["ios", "android"],
+  "ios": {
+    "modules": ["SymbolModule"]
+  },
+  "android": {
+    "modules": ["expo.modules.symbols.SymbolModule"]
+  }
```

```suggestion
  }
```

do we support android?